tempora/schedule.py

```diff
@@ -176,7 +186,7 @@ def run_pending(self):
             del self.queue[0]
 
     @abc.abstractmethod
-    def run(self, command):
+    def run(self, command) -> None:
```

Comment:
These commands are meant to be DelayedCommands.

```suggestion
    def run(self, command: DelayedCommand) -> None:
```